apps/acrs/behaviours/declaration.js

```diff
@@ -1,6 +1,7 @@
 const Utilities = require('../../../lib/utilities');
+const config = require('../../../config');
 
-const CheckUnder18InAggregatedValue = (aggregateArray, fieldLabel) => {
+const CheckUnder18InAggregatedValue = (aggregateArray, fieldLabel, format, ageLimit) => {
```

Comment:
You don't really need to pass format and ageLimit into this function as they are available and always the same in the upper scope, but I can see how it might be useful to do so if the values were different between children, siblings etc.

I see benefits in both approaches so no complaints, but again you can make the parameters of the calling of this function neater by allowing it to use values from the upper scope instead.

Reply:
I understand what you mean but adding it in higher level will result in tight coupling. if you get rid of the file or property at one point the method depending on it will need refactoring providing parameters at lower level can help in useable code

Reply:
Yes I hear you, I'm not keen on drilling those values through, but I guess you'd have to pull the whole thing apart to improve that. All good